.circleci/config.yml

```diff
@@ -104,8 +104,11 @@ jobs:
                 curl -s https://api.github.com/repos/os-autoinst/os-autoinst/commits | grep sha | head -n 1 | grep -o -E '[0-9a-f]{40}' > .circleci/autoinst.sha
                 git add .circleci/dependencies.txt .circleci/autoinst.sha
                 git commit -m "Dependency cron $depid"
-                git push $CIRCLE_REPOSITORY_URL dependency_$depid
-                hub pull-request -m "Dependency cron $depid" --base $CIRCLE_PROJECT_USERNAME:$CIRCLE_BRANCH
+                echo 'echo $GITHUB_TOKEN' > $(pwd)/.git-askpass
+                export GIT_ASKPASS=$(pwd)/.git-askpass
+                chmod +x $(pwd)/.git-askpass
+                git push -q -f https://token@github.com/openqabot/openQA.git dependency_$depid
```

Comment:
What repo is that?

Reply:
Obviously the bot's fork: https://github.com/openqabot/openQA

It make sense considering the bot creates a PR.

Reply:
Ah, it does when you put it like that. If it has its own fork, maybe it should also get back its own email ie. skynet@open.qa?

Reply:
Or you ask nicely... :) https://www.skynet.net/contact